src/BoilerPlate/App.cpp

```diff
@@ -130,7 +200,15 @@ namespace Engine
 
 		// Update code goes here
 		//
+		m_player->Update(DESIRED_FRAME_TIME);
+		//m_asteroid->Update(DESIRED_FRAME_TIME);
+
+		std::vector<Asteroid>::iterator it = m_asteroids.begin();
 
+		for (; it != m_asteroids.end(); it++)
+		{
+			(*it).Update(DESIRED_FRAME_TIME);
```

Comment:
Remember to change this call for `deltatime`. You'll need it for the rendering plot.